mwdb/core/config.py

```diff
@@ -60,7 +60,8 @@ class MWDBConfig(Config):
     base_url = key(cast=str, required=False, default="http://127.0.0.1")
     # Flask additional settings file (optional)
     flask_config_file = key(cast=path, required=False)
-
+    # Axios request timeout
+    request_timeout = key(cast=int, required=False, default=8000)
```

Comment:
Default value doesn't match with documentation
```suggestion
    request_timeout = key(cast=int, required=False, default=15000)
```

Reply:
Sorry, it was my fault. Request_timeout should be 20000 ms as default value and of course in documentation. It should be 20000 because default statement_timeout = 15000. Request_timeout should be greater than statement_timeout. It will be corrected.